KanBanApi/KanBanApi.Application/Dtos/TodoResponse.cs

```diff
@@ -0,0 +1,3 @@
+namespace KanBanApi.Application.Dtos;
+
+public record TodoResponse(string Id, string Status, string Title, string Message, DateOnly? DueDate, string? List);
```

Comment:
Why is this named `TodoResponse`. Isn't this the dto returned after creating a new todo? Why is it not called `NewTodoResponse` ?